### ISSUE TITLE
[Bug Report][3.1.2] VChip without aria-label for closable chips

### ISSUE THREAD

**Issue by mediafreakch** [vuetifyjs/vuetify#16549]
### Environment
**Vuetify Version:** 3.1.2
**Last working version:** 2.6.13
**Vue Version:** 3.2.45
**Browsers:** Chrome 109.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
1. Open the Stackblitz example attached, or inspect a closable v-chip on the official vuetify docs using DevTools

### Expected Behavior
- Chip's close icon button has an aria-label
- Aria-label's value is the value of `$vuetify.close`  in the default language for the vuetify internationalization (even if no internationalization configured in theme)
- Using `close-label` prop, one can define a custom value for the aria-label

### Actual Behavior
- Chip's close icon button does not have an aria-label
- Setting a locale in the theme does not make the aria-label work
- Setting the `close-label` prop has no effect

### Reproduction Link
[https://stackblitz.com/edit/vitejs-vite-fzvzcm?file=package.json](https://stackblitz.com/edit/vitejs-vite-fzvzcm?file=package.json)
